docs/data-sources/instance_ip.md

```diff
@@ -0,0 +1,35 @@
+---
+page_title: "Scaleway: scaleway_instance_ip"
+description: |-
+  Gets information about an instance IP.
+---
+
+# scaleway_instance_ip
+
+Gets information about an instance IP.
+
+## Example Usage
+
+```hcl
+# Get info by IP address
+data "scaleway_instance_ip" "my_ip" {
+  address = "0.0.0.0"
+}
+```
+
+## Argument Reference
+
+- `address` - (Required) The IP address.
```

Comment:
Could you add getting an IP from its ID?

Reply:
Yes working on it